mandatory/1-fix-functions.js

```diff
@@ -10,8 +10,8 @@
 
 */
 
-function mood() {
-  let isHappy = true;
+function mood(today) {
+  let isHappy = today;
 
   if (isHappy) {
```

Comment:
slight optimization here; if `isHappy === today` you could just remove the `isHappy` declaration and just pass `today` into the `if()` statement